examples/regression/script.jl

```diff
@@ -0,0 +1,196 @@
+# A recreation of <https://gpflow.readthedocs.io/en/master/notebooks/advanced/gps_for_big_data.html>
+
+# # Stochastic Variational Regression
+#
+# In this example, we show how to construct and train the stochastic variational
+# Gaussian process (SVGP) model for efficient inference in large scale datasets.
+# For a basic introduction to the functionality of this library, please refer to
+# the [User Guide](@ref).
+#
+# ## Setup
+
+using SparseGPs
+using Distributions
+using LinearAlgebra
+using IterTools
```

Comment:
what are these used for? maybe an explicit
```suggestion
using IterTools: ncycle
```
(or whichever it is) could be helpful here